bindings_ffi/src/mls.rs

```diff
@@ -947,6 +947,26 @@ impl FfiConversations {
         Ok(convo_list)
     }
 
+    pub async fn list_conversations(
```

Comment:
I'm not sure I understand when this is supposed to be used instead of the `list()` function above?

Reply:
What I saw in android, we fetched the conversations and the get the last message one by one and map them together. 
we can replace it with this.